architecture/additional_concepts/storage.adoc

```diff
@@ -20,11 +20,11 @@ specific knowledge of the underlying storage infrastructure.
 
 PVCs are specific to a
 link:../../architecture/core_concepts/projects_and_users.html#project[project]
-and are created and used by developers. A PVC is required if a developer wants
-to use a PV. The PVs themselves are not bound to a single project; they can be
-shared across the entire OpenShift cluster and claimed from any project. After a
-PV has been bound to a PVC, however, that PV cannot be bound to additional PVCs.
-This has the effect of scoping a bound PV to a single
+and are created and used by developers as a means to use a PV. PV resources on
+their own are not scoped to any single project; they can be shared across the
+entire OpenShift cluster and claimed from any project. After a PV has been
+link:#binding[_bound_] to a PVC, however, that PV cannot then be bound to
+additional PVCs. This has the effect of scoping a bound PV to a single
 link:../../architecture/core_concepts/projects_and_users.html#namespaces[namespace]
```

Comment:
Hi Alex,  I was using the github rendering (I forgot). My mistake...


----- Original Message -----
> > @@ -20,11 +20,11 @@ specific knowledge of the underlying storage
> > infrastructure.
> >  
> >  PVCs are specific to a
> >  link:../../architecture/core_concepts/projects_and_users.html#project[project]
> > -and are created and used by developers. A PVC is required if a developer
> > wants
> > -to use a PV. The PVs themselves are not bound to a single project; they
> > can be
> > -shared across the entire OpenShift cluster and claimed from any project.
> > After a
> > -PV has been bound to a PVC, however, that PV cannot be bound to additional
> > PVCs.
> > -This has the effect of scoping a bound PV to a single
> > +and are created and used by developers as a means to use a PV. PV
> > resources on
> > +their own are not scoped to any single project; they can be shared across
> > the
> > +entire OpenShift cluster and claimed from any project. After a PV has been
> > +link:#binding[_bound_] to a PVC, however, that PV cannot then be bound to
> > +additional PVCs. This has the effect of scoping a bound PV to a single
> >  link:../../architecture/core_concepts/projects_and_users.html#namespaces[namespace]
> 
> Are you looking at the GitHub render of the doc? It won't respect ifdefs. It
> should be visible in the built versions of the docs:
> 
> http://file.rdu.redhat.com/~adellape/031016/storage-sec-update/architecture/additional_concepts/storage.html#types-of-persistent-volumes
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/openshift-docs/pull/1726/files/e2a53d732c95646e2a5cab715ca86f9edae5019f..2a8f749d67480f3056c4973e786bc4204d8ef3d2#r57059097